libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SalesforceHybridSDKManager.h

```diff
@@ -31,7 +31,7 @@ NS_ASSUME_NONNULL_BEGIN
  */
 
 @interface SalesforceHybridSDKManager : SmartSyncSDKManager
-
+@property (nonatomic, strong, nullable) SFSDKAppConfig *appConfig;
```

Comment:
This property exists on `SalesforceSDKManager`.  We don't need to redefine it, right?

Reply:
@khawkins you are right. Should be removed.

Reply:
This can be removed.